diagnostic.md

```diff
@@ -5,19 +5,20 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What is the purpose of a backend?
 
 ```md
-// your response here
+It takes user requests, routes them, processes them, and returns them to the user.
+It stores all the models, dbs, methods, etc.
```

Comment:
Most importantly it stores data and persists it from user to user and between sessions.